test/JDBC/expected/BABEL-1442.out

```diff
@@ -239,6 +235,10 @@ GO
 
 CREATE LOGIN r4 FROM WINDOWS;
 GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The Server principal 'r4' already exists)~~
```

Comment:
Does T-SQL even accept this login name format for windows logins?

Reply:
We are going to track this separate task BABEL-3828